test/python/test_roles_api.py

```diff
@@ -801,7 +801,8 @@ def test_parameter_combos_b(self):
         ]
 
         self.assertEqual(status, 200)
-        self.assertEqual(details, target_details)
+        for i in target_details:
+            self.assertIn(i, details)
```

Comment:
I'm not sure these are the same conditions - you could have every element of `target_details` in `details`, but still have some elements from `details` that are missing from `target_details` - right? is there something about these arrays that you can use to determine this check is sufficient (and maybe a comment would be good in that case), or are there other things you also need to check (like all elements are unique, and the size of the two arrays are the same)?

Reply:
You are correct, it was not quite as robust as it needed to be. I updated the check, now it will fail if the two lists differ in length, or have any elements which are not present in the other.